src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -61,7 +62,7 @@ export class TaskListBuilder {
       )
     }
 
-    if (draft.isResponsePartiallyRejectedDueTo(RejectPartOfClaimOption.PAID_WHAT_BELIEVED_WAS_OWED)) {
+    if (draft.requirePartialAdmitHandOff()) {
```

Comment:
As I said earlier, it does not make sense to me to have it when hand off scenario is triggered.